### PR TITLE
Add strimzi category to our CRDs

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

### Type of change
- Enhancement

### Description
Fixes https://github.com/strimzi/strimzi-kafka-operator/issues/2427
We can get all strimzi CRs by `kubectl get strimzi`, for example
```
[sknot@sknot strimzi-kafka-operator]$ kubectl get strimzi
NAME       PARTITIONS   REPLICATION FACTOR
my-topic   1            1

NAME         DESIRED KAFKA REPLICAS   DESIRED ZK REPLICAS
my-cluster   1                        1

NAME      AUTHENTICATION   AUTHORIZATION
my-user   tls              simple

NAME                  AGE
my-source-connector   21s

NAME        DESIRED REPLICAS
my-bridge   1
```

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: api/src/main/java/io/strimzi/api/kafka/Crds.java [hunk 1]
```diff
@@ -43,6 +43,7 @@
 public class Crds {
 
     public static final String CRD_KIND = "CustomResourceDefinition";
+    public static final String CRD_CATEGORY = "strimzi";
```
Comment (tombentley) [inline api/src/main/java/io/strimzi/api/kafka/Crds.java]: I'd call this `STRIMZI_CATEGORY`. I'm also not sure this is really the correct class to declare it on. Though I can't think of a better one. Looking at our CRD-annotated classes, such as `Kafka`, we have various constants which are common across numerous classes:

public static final String CRD_API_VERSION = "apiextensions.k8s.io/v1beta1";
public static final String RESOURCE_GROUP = "kafka.strimzi.io";

But I'm not sure even that's enough to warrant some `CrdConstants` class. Thoughts @scholzj ?

Comment (stanlyDoge) [reply api/src/main/java/io/strimzi/api/kafka/Crds.java]: Tested with both `oc` and `kubectl`

Comment (ppatierno) [reply api/src/main/java/io/strimzi/api/kafka/Crds.java]: Imho maybe we could have a CRD related constants class but if we do it in this PR we'll have a lot of unrelated changes.

Comment (stanlyDoge) [reply api/src/main/java/io/strimzi/api/kafka/Crds.java]: I've opened an issue for this: https://github.com/strimzi/strimzi-kafka-operator/issues/2437

**Pull request merged**